pkg/dynamic/discovery/discovery.go

```diff
@@ -99,8 +99,11 @@ func (rm *ResourceMap) refresh() {
 	logging.Logger.V(7).Info("Refreshing API discovery info")
 	_, groups, err := rm.discoveryClient.ServerGroupsAndResources()
 	if err != nil {
-		logging.Logger.Error(err, "Failed to fetch discovery info")
-		return
+		if len(groups) == 0 {
+			logging.Logger.Error(err, "Failed to fetch discovery info")
+			return
+		}
+		logging.Logger.Error(err, "Failed to fetch all resources, continuing with partial discovery info")
```

Comment:
nit: Will it be good to log this as INFO perhaps with a certain log level?
query: Is it possible to figure out the resources that failed to discover?

Reply:
Hi @AmitKumarDas , yes I think info makes more sense. I can update that 👍  Regarding figure out the resources that failed discovery, the error that is logged contains that info.
For example,
```error":"unable to retrieve the complete list of server APIs: external.metrics.k8s.io/v1beta1: the server is currently unable to handle the request"```
thanks for the feedback!

Reply:
Thanks @mjsmith1028